Dockerfile

```diff
@@ -1,11 +1,37 @@
-FROM golang:1.18 AS builder
+FROM --platform=${BUILDPLATFORM} golang:1.21 AS builder
 
-WORKDIR /go/src/github.com/apecloud/open-local
-COPY . .
```

Comment:
which is best practises (1. prebuild and just copy, 2. build from source code then copy)?